presto-cassandra/src/main/java/io/prestosql/plugin/cassandra/NativeCassandraSession.java

```diff
@@ -317,18 +318,34 @@ private static void checkColumnNames(List<ColumnMetadata> columns)
         }
     }
 
-    private CassandraColumnHandle buildColumnHandle(AbstractTableMetadata tableMetadata, ColumnMetadata columnMeta, boolean partitionKey, boolean clusteringKey, int ordinalPosition, boolean hidden)
+    private Optional<CassandraColumnHandle> buildColumnHandle(AbstractTableMetadata tableMetadata, ColumnMetadata columnMeta, boolean partitionKey, boolean clusteringKey, int ordinalPosition, boolean hidden)
     {
-        CassandraType cassandraType = CassandraType.getCassandraType(columnMeta.getType().getName());
+        Optional<CassandraType> cassandraType = CassandraType.getCassandraType(columnMeta.getType().getName());
+        if (!cassandraType.isPresent()) {
+            log.debug("Unsupported column type: " + columnMeta.getType().getName());
```

Comment:
`log.debug("Unsupported column type: %s",  columnMeta.getType().getName())` (avoid string formatting when debug log is not enabled)